src/Models/Channel.cs

```diff
@@ -50,6 +50,9 @@ public class Channel : CustomDataBase
 
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "last_message_at")]
         public DateTimeOffset? LastMessageAt { get; set; }
+
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "truncated_by")]
+        public User TruncatedBy { get; set; }
```

Comment:
you can get `truncated_by_id` if it's a deleted user

Reply:
The backend doesn't return TruncatedById:

https://github.com/GetStream/chat/blob/ba899d26f0ab98b0ad9b5be7f591a9aaa651c2eb/types/channel.go#L803=

Reply:
seems we don't expose it due to failing enrichment, we should have done though in ideal world